### ISSUE TITLE
How are we handling accessibility?

### ISSUE THREAD

**Issue by suzannehillman** [DevProgress/maps-showcase#9]
There are a number of possibly accessibility concerns for this project. The ones that came to mind include: 

Captioning for videos, and any other audio-only things should be transcribed. 

For blind and vision impaired, anything that's shown in a picture or video needs to be made available through text or audio means, and the site in general needs to be made to support blind and vision impaired people. Eg, section 508, WCAG, using Aria and other useful tags.

**Comment by deborahgu** [comment]
1. [Autoplay videos with audio discouraged under WCAG](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html)
2. [Captions and transcripts](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html)
3. [Keyboard controls for all video players, sliders, slideshows, etc., are required.](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html) For some video embeds (e.g. YouTube, Kaltura) this is doable with a little bit of WAI-ARIA massaging. Others need a lot more handholding. 
4. All images, including maps and pure CSS sprites, need appropriate alternative text (or null alt for decorative). If the page isn't meaningful non-visually, a non-visual alternative, best as possible, should be provided.
5. Color contrast should pass basic checks.
6. Site should be navigable by screen reader and with a keyboard.
7. Site should have a skip-to-main-content link which appears on tab-to-focus.

**Comment by racoonsmakeokayfriends** [comment]
For 5, we can also check for color blindness

**Comment by jillh510** [comment]
Should this issue be marked "help wanted"?
